packages/algolia/src/indexes/research-output.ts

```diff
@@ -0,0 +1,31 @@
+import { WaitablePromise } from '@algolia/client-common';
+import { DeleteResponse, SaveObjectResponse } from '@algolia/client-search';
+
+import { ResearchOutputResponse } from '@asap-hub/model';
+import { SearchIndex } from './search-index';
+import { SearchResponse } from '../types/response';
+
+export type ResearchOutputSearchResponse = SearchResponse<
+  ResearchOutputResponse,
+  'research-output'
+>;
+
+export class ResearchOutputSearchIndex extends SearchIndex<
```

Comment:
is it possible to create a `fetch` method here and use the package to fetch the data in FE? otherwise this does not need to be a package as it is only used by asap-server

At the moment the FE API is detached from the types:
https://github.com/yldio/asap-hub/pull/1281/files#diff-c307e5f0df5297cb25db5b6ef5d0683f149ba7ce5279271666870227c27fde4dL72